docs/SRS/SRS.tex

```diff
@@ -59,21 +60,69 @@ \subsection{Goals of the Project}
 \lips
 \section{Stakeholders}
 \subsection{Client}
-\lips
+The client of this project is \textbf{Dr. Istvan David} from McMaster's Department of Computing and Software. As the project supervisor, his role is to guide the development team with his technical and domain expertise. As the client, he sets the product's requirements and will be involved throughout its development. 
 \subsection{Customer}
-\lips
+The customers of this product are all the \textbf{software developers} that use it to improve the energy efficiency of their codebase. They will be the primary users of the product and, therefore, will offer critical feedback on its effectiveness. Suggestions for improvement and/or additional features may come from this stakeholder. 
```

Comment:
Kinda reads like "the customers of this product are the users" might be good to write that the customers r probably early career software developers who need help restructuring code. but obv thats not all-encompassing so I see why you didn't do it.